library/src/main/java/com/felipecsl/gifimageview/library/GifHeaderParser.java

```diff
@@ -264,6 +264,9 @@ private void readNetscapeExt() {
         int b1 = ((int) block[1]) & 0xff;
         int b2 = ((int) block[2]) & 0xff;
         header.loopCount = (b2 << 8) | b1;
+        if(header.loopCount > 0) {
+          header.loopCount++;
```

Comment:
This was admittedly confusing so I have updated to clarify in 2855541125f0395161ab4d1bd969c9d7aa44087b. The issue is that after testing various files I found that the "NETSCAPE2.0" header will not exist at all in the case that the gif should only play once but a value of zero is used to indicate loop infinitely. I have updated now so that the value of zero is sniffed out and then set to -1 for loop infinitely which allows a default value of 0 to indicate that the file should be played once and not loop at all.

So for clarity the 3 cases for this loopCount header are now:
1. Netflix is present in header and set to 0 :: header.loopCount is set to -1 and the gif will loop infinitely
2. Netflix is not present in header :: header.loopCount will default to 0 and gif will loop once
3. Netflix is present in header and > 0  :: header.loopCount is explicitly set to the value indicated in the header